apis/notifications.md

```diff
@@ -129,6 +129,71 @@ Build and run your app again, and press the "Notify" button. Notice that the not
 Remember that `SimpleAction`s added in the `Application` class with `add_action ()` are automatically added in the `app` namespace. Notifications can't trigger actions defined in other namespaces like `win`.
 {% endhint %}
 
+## Default Action
+
+You may have noticed that when you click on a new notification, a new window pops up. This is happening because the notification has a default action that is executed when the user clicks on it. If you don't set it, it will `activate` your application, where we create a new window and present it.
+
+You can change the default action using  [`set_default_action ()`](https://valadoc.org/gio-2.0/GLib.Notification.set\_default\_action.html)
+
+```vala
+notify_button.clicked.connect (() => {
+    var notification = new Notification ("Hello World");
+    notification.set_body ("This is my first notification!");
+    notification.set_default_action ("app.quit");
+
+    send_notification (null, notification);
+});
+```
+
+If you want to avoid creating a new window every time your application is activated, you need to store your main window in an instance variable and check if it has already been shown.
```

Comment:
I wonder if this should be a different section in the docs about single window vs multi-window app. There's other things related to this to mention like adding the launcher key for single window app and how the dock handles this

Reply:
Hmm... I think this is a bit out of scope of this branch. Mind if I add this section in a separate branch? It's easy to write about single window apps but I'm not familiar with handling multiple windows so I'll need to look at how Files handle this for example.